pymobiledevice3/services/webinspector.py

```diff
@@ -8,7 +8,8 @@
 
 import nest_asyncio
 
-from pymobiledevice3.exceptions import WebInspectorNotEnabledError, RemoteAutomationNotEnabled
+from pymobiledevice3.exceptions import WebInspectorNotEnabledError, RemoteAutomationNotEnabled, \
```

Comment:
`WebInspectorNotEnabledError` and `RemoteAutomationNotEnabled` does not exist in this commit